painttiming.bs

```diff
@@ -62,9 +89,42 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+A [=generated content pseudo-element=] is a <dfn>paintable pseudo-element</dfn> when all of the following apply:
+* The pseudo-element's [=used=] [=visibility=] is <code>visible</code>.
+* The pseudo-element's [=used=] [=opacity=] is greater than zero.
+* The pseudo-element generates a non-empty [=box=].
+
+An [=element=] is <dfn export>contentful</dfn> when one or more of the following apply:
```

Comment:
Would be cleaner to create a variable here and refer to it further down. e.g. "An [=element=] |el| is..." and then replace "The element" with "|el|"

Reply:
ok